ocp_resources/csi_storage_capacity.py

```diff
@@ -32,7 +32,7 @@ def __init__(
         self.storage_class_name = storage_class_name
         self.maximum_volume_size = maximum_volume_size
 
-    def to_dict(self):
```

Comment:
should the __init__() be updated at some point as well? I know it would be too much for this one.

Reply:
everything should be updated at some point.